### ISSUE TITLE
Awkward to recover after manually deleting workspace dir

### ISSUE THREAD

**Issue by cs-shadow** [cs-shadow/buildstream#84]
In GitLab by @aevri on Aug 31, 2017, 09:51

If I do this:

```
bst workspace open myelement.bst ../myelement
rm -rf ../myelement
```

I can't use `bst` to close the workspace:

```
bst workspace close myelement.bst
Loading:   020
Resolving: 020/020
Error loading pipeline: Failed loading workspace. Did you remove the workspace directory? [Errno 2] No such file or directory: '/src/myelement'
```

To recover, I can `mkdir ../myelement`, and then close the workspace. It feels a bit awkward to have to do so though, I simply want to remove it :)

**Comment by cs-shadow** [comment]
In GitLab by @tristanvb on Sep 4, 2017, 19:03

closed via commit fe9a88a01347b729e0a928ea188f6cfc51f00d02

**Comment by cs-shadow** [comment]
In GitLab by @tristanvb on Sep 5, 2017, 21:31

mentioned in commit fe9a88a01347b729e0a928ea188f6cfc51f00d02
